daemon/delete.go

```diff
@@ -55,7 +57,19 @@ func (daemon *Daemon) ContainerRm(job *engine.Job) engine.Status {
 		if container.IsRunning() {
 			if forceRemove {
 				if err := container.Kill(); err != nil {
-					return job.Errorf("Could not kill running container, cannot remove - %v", err)
+					if err == execdriver.ErrProcessNotFound {
+						// This is a situation in which the container PID 1 has died
+						// but has yet to be reaped or recognized by Docker as being gone.
+						// wait for the container to stop
+						if _, err := container.WaitStop(10 * time.Second); err != nil {
```

Comment:
I can change it to one second.  The 10 seconds was just copied from Container.Kill()